style.user.css

```
@@ -720,11 +720,16 @@ background-color: draculaW-blackSecondary
     }
   }
-
+  
+  /* Color the topbar */
+  .AppHeader {
+      background-color: var(--color-header-bg);
+  }
+  
   /* Color the logo */
   .octicon-mark-github {
     if purple-logo {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)